decidim-proposals/lib/decidim/proposals/markdown2proposals.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+require "redcarpet"
+
+module Decidim
+  module Proposals
+    # This class parses a participatory text document in markdown and
+    # produces Proposalsin the form of sections and articles
```

Comment:
Please add some docs about how Redcarpets works and which/when the methods are called.